Util/Csrf/CsrfCheckerTrait.php

```diff
@@ -13,16 +16,14 @@ trait CsrfCheckerTrait
      * @param string $id    The id used when generating the token
      * @param string $query
      */
-    protected function checkCsrf($id = 'lexik-translation', $query = '_token')
+    protected function checkCsrf(Request $request, ?CsrfTokenManager $tokenManager, $id = 'lexik-translation', $query = '_token')
```

Comment:
Instead of adding 2 new parameters to this method, I think you should add `$requestStack` & `$tokenManager` properties to this trait.

Plus add 2 methods in the trait to set them: `setRequestStack()` & `setTokenManager()` using `@required` annotation.

See https://symfony.com/doc/current/service_container/autowiring.html#autowiring-other-methods-e-g-setters-and-public-typed-properties

This will be more clean than passing 2 params everywhere.

Reply:
@Seb33300 Done